taichi/program/compile_config.h

```diff
@@ -33,6 +33,7 @@ struct CompileConfig {
   bool fast_math;
   bool use_unified_memory;
   bool async;
+  bool no_cp2o;
```

Comment:
If `ArgLoadStmt` is used, this will become unnecessary. I think this boolean variable is not a good design.

Of course, feel free to use these expedient variables for debugging before merging into master :)